_posts/2021-04-08-pulp-3.12-is-generally-available.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Pulp 3.12.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+Let's take a look at the main updates and enhancements as part of this release.
+
+This release also includes a plethora of bugfixes as well as some nice enhancements to Pulpcore and the Plugin API. For a full overview of changes, check out the [changlog](https://docs.pulpproject.org/pulpcore/changes.html).
+
+## Support added for automatic publishing and distributing [#7626](https://pulp.plan.io/issues/7626)
+
+When the content of a repository is updated, the repository can now be automatically published and redestributed.
+
+## New Checksum validation steps
+
+Pulp can prohibit or allow checksums by setting the `ALLOWED_CONTENT_CHECKSUMS` setting.
+
+As part of this release, a number of validation steps have been added to help to ensure that you can debug and manage content checksums that are synched as part of the on-demand sync process.
+
+New documenation has been also added for the [ALLOWED_CONTENT_CHECKSUMS setting](https://docs.pulpproject.org/pulpcore/en/master/nightly/settings.html#allowed-content-checksums).
+
+### Checking remote artifacts against ALLOWED_CONTENT_CHECKSUMS [#7985](https://pulp.plan.io/issues/7985)
+
+A new warning has been added to the on-demand sync process to check if remote artifacts are detected with checksums that do not correspond with the `ALLOWED_CONTENT_CHECKSUMS`.
+
+### New sync check for forbidden checksums [#8423](https://pulp.plan.io/issues/8423)
+
+During a sync of on-demand content, a new check is performed and throws an error if forbidden checksums are detected.
+
+As part of [#7989](https://pulp.plan.io/issues/7989) more enhancements updates were made to handle the exception that gets raised on the client side.
```

Comment:
Sounds very similar to me. Can we collapse those two issues into one announcement?

Reply:
Removed the headings and just kept the initial headline.